<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 默认 */
QToolButton{   
	border-top: 3px outset transparent;           /* 上边框宽度3像素、突出显示、颜色为透明 */
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 80px;                            /* 控件最小宽度 */
    min-height: 80px;
	background-color: rgb(228, 228, 228);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97); /* 左边框宽度3像素、突出显示、颜色为深深灰 */
	background-color: rgb(246, 246, 246);
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="sideBar_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="sideBar">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QToolButton" name="monitorButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">/* 默认 */
QToolButton{   
	border-top: 3px outset transparent;           /* 上边框宽度3像素、突出显示、颜色为透明 */
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 80px;                            /* 控件最小宽度 */
    min-height: 80px;
	background-color: rgb(228, 228, 228);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97); /* 左边框宽度3像素、突出显示、颜色为深深灰 */
	background-color: rgb(246, 246, 246);
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
       </property>
       <property name="text">
        <string>Monitor</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="connectionButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>  Connection</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="consoleButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">/* 默认 */
QToolButton{   
	border-top: 3px outset transparent;           /* 上边框宽度3像素、突出显示、颜色为透明 */
	border-bottom: 7px outset transparent;
	border-right: 3px outset transparent;
	border-left: 3px outset transparent;
    min-width: 80px;                            /* 控件最小宽度 */
    min-height: 80px;
	background-color: rgb(228, 228, 228);
}

/* 鼠标悬停 */
QToolButton:hover{
	background-color: rgb(205, 205, 205);
}

/* 点击和按下 */
QToolButton:pressed,QToolButton:checked{
	border-left: 3px outset rgb(93, 95, 97); /* 左边框宽度3像素、突出显示、颜色为深深灰 */
	background-color: rgb(246, 246, 246);
}

QPushButton:default {
    border-color: navy; /* make the default button prominent */
}</string>
       </property>
       <property name="text">
        <string>Console</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>4</y>
      <width>941</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
